drivers/include/atwinc15x0.h

```diff
@@ -54,16 +63,30 @@ typedef struct {
     gpio_t wake_pin;        /**< WAKE pin */
 } atwinc15x0_params_t;
 
+/**
+ * @brief   ATWINC15x0 internal states
+ */
+typedef enum {
+    ATWINC15X0_STATE_SLEEP,                     /**< Sleep state */
+    ATWINC15X0_STATE_DISCONNECTED,              /**< Disconnect state */
+    ATWINC15X0_STATE_DISCONNECTED_SCANNING,     /**< Scanning state when disconnected */
+    ATWINC15X0_STATE_DISCONNECTED_CONNECTING,   /**< Connecting state where disconnected before */
+    ATWINC15X0_STATE_CONNECTED,                 /**< Connected state */
+    ATWINC15X0_STATE_CONNECTED_SCANNING,        /**< Scanning state when connected */
+    ATWINC15X0_STATE_CONNECTED_CONNECTING,      /**< Connecting state where disconnect event is not yet received */
```

Comment:
Exceeding the maximum line length of 100 characters
```suggestion
    ATWINC15X0_STATE_CONNECTED_CONNECTING,      /**< Connecting state where disconnect event is not yet received */
```